types/vimeo__player/index.d.ts

```diff
@@ -367,7 +397,7 @@ export interface Options {
     controls?: boolean | undefined;
     dnt?: boolean | undefined;
     height?: number | undefined;
-    interactiveparams?: string | undefined;
+    interactive_params?: string | undefined;
```

Comment:
This was originally misspelled... it was fixed in the latest version:
https://github.com/vimeo/player.js/pull/811